src/views/Communities/Tabs.tsx

```diff
@@ -54,12 +54,12 @@ const TabsComponent = ({setStatusFilter, setActiveTab, activeTab, statusFilter,
         }>
             <TabList>
                 <Tab
-                    onClick={() => {setActiveTab('all'); update('type', 'all') }}
+                    onClick={() => {setActiveTab('all'); update('type', 'all'); setItemOffset(0) }}
```

Comment:
Create a method for this (try to do the same every-time you need to run more than a fn.